src/SFA.DAS.DATA.MANAGEMENT/Tables/AsDataPL_Comt_Apprenticeship.sql

```diff
@@ -28,6 +28,7 @@
 	[MadeRedundant] [bit] NULL,
 	[OriginalStartDate] [datetime] NULL,
 	[Age] [int] NULL,
+	[Email]  [NVarchar](200) NULL,
```

Comment:
Is it a requirement to bring Apprentice Email. That's a sensitive information.

Reply:
Taken off email for Apprenticeship as discussed